sdk/nodejs/package.json

```diff
@@ -21,8 +21,10 @@
         "@opentelemetry/semantic-conventions": "^1.6.0",
         "@pulumi/query": "^0.3.0",
         "@types/google-protobuf": "^3.15.5",
+        "@types/tmp": "^0.2.6",
         "execa": "^5.1.0",
         "google-protobuf": "^3.5.0",
+        "got": "^11.8.6",
```

Comment:
This version of got is the most recent that still supports node 10.

The nodejs SDK package.json has:

```
    "engines": {
        "node": ">=8.13.0 || >=10.10.0"
    },
```